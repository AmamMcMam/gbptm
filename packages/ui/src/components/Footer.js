import React, { Component } from 'react';
import { Link } from 'react-router';

import styles from './css/footer.module.css';

class Footer extends Component {
  render() {
    return (
      <footer className={styles.footer}>
        <ul className={styles.footerList}>
          <li className={styles.footerListItem}>
            <Link to="/preferences" className={styles.footerItem}>
              <span className={styles.footerItemText}>
                My toilet preferences
              </span>
            </Link>
          </li>
          <li className={styles.footerListItem}>
            <Link to="/about" className={styles.footerItem}>
              <span className={styles.footerItemText}>About this project</span>
            </Link>
          </li>
        </ul>
      </footer>
    );
  }
}

export default Footer;
