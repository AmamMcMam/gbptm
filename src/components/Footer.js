import React from 'react';
import PropTypes from 'prop-types';
import { Link, withRouter } from 'react-router-dom';

import styles from './css/footer.module.css';

import CookieBoxButton from './Tracking/CookieBoxButton';

const chooseLinkStyle = (location, path) => {
  if (location.pathname === path) {
    return styles.footerItemActive;
  }

  return styles.footerItem;
};

const Footer = (props) => (
  <footer className={styles.footer}>
    <ul className={styles.footerList}>
      <li className={styles.footerListItem}>
        <Link
          to="/privacy"
          className={chooseLinkStyle(props.location, '/privacy')}
        >
          <span className={styles.footerItemText}>Privacy Policy</span>
        </Link>
      </li>
      <li className={styles.footerListItem}>
        <Link to="/about" className={chooseLinkStyle(props.location, '/about')}>
          <span className={styles.footerItemText}>About</span>
        </Link>
      </li>
      <li className={styles.footerListItem}>
        <CookieBoxButton
          onClick={props.onCookieBoxButtonClick}
          isCookieSettingsOpen={props.isCookieSettingsOpen}
        />
      </li>
    </ul>
  </footer>
);

Footer.displayName = 'Footer';
Footer.propTypes = {
  onCookieBoxButtonClick: PropTypes.func.isRequired,
  isCookieSettingsOpen: PropTypes.bool,
  location: PropTypes.shape({
    pathname: PropTypes.string,
  }),
};

export default withRouter(Footer);
