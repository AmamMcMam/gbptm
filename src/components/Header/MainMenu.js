import React from 'react';
import PropTypes from 'prop-types';
import Link from 'next/link';
import { useRouter } from 'next/router';
import styled from '@emotion/styled';

import Box from '../Box';
import Text from '../Text';
import { Media } from '../Media';
import { useAuth } from '../../Auth';

const StyledNavLink = styled(Link)`
  // active class is added by NavLink component
  &.active {
    color: ${(props) => props.theme.colors.tertiary};
  }
`;

// Todo: Contact link
const MainMenu = ({
  mapCenter,
  onMenuItemClick = Function.prototype,
  children,
}) => {
  const { isAuthenticated, logout } = useAuth();
  const history = useRouter();

  const handleLogout = () => {
    logout();
    onMenuItemClick();
    history.push('/');
  };

  return (
    <Text
      fontWeight="bold"
      textAlign={['center', 'left']}
      css={{
        // fill container on mobile
        height: '100%',
      }}
    >
      <Box
        display="flex"
        flexDirection={['column', 'row']}
        height={['100%', 'auto']}
      >
        <Box
          as="ul"
          display="flex"
          flexDirection={['column', 'row']}
          height={['100%', 'auto']}
          width="100%"
        >
          <Box as="li" ml={[0, 4]}>
            <StyledNavLink href="/" onClick={onMenuItemClick} exact>
              Find Toilet
            </StyledNavLink>
          </Box>
          <Box as="li" mt={[3, 0]} ml={[0, 4]}>
            <StyledNavLink
              href={
                mapCenter
                  ? `/loos/add?lat=${mapCenter.lat}&lng=${mapCenter.lng}`
                  : `/loos/add`
              }
              onClick={onMenuItemClick}
            >
              Add Toilet
            </StyledNavLink>
          </Box>

          <Box as="li" mt={['auto', 0]} ml={[0, 'auto']}>
            <StyledNavLink href="/about" onClick={onMenuItemClick}>
              About
            </StyledNavLink>
          </Box>
          <Box as="li" mt={[3, 0]} ml={[0, 4]}>
            <StyledNavLink href="/use-our-loos" onClick={onMenuItemClick}>
              Our Sponsor
            </StyledNavLink>
          </Box>
          <Box as="li" mt={[3, 0]} mb={['auto', 0]} ml={[0, 4]}>
            <StyledNavLink href="/contact" onClick={onMenuItemClick}>
              Contact
            </StyledNavLink>
          </Box>

          {isAuthenticated() && (
            <Box as="li" mt={[3, 0]} mb={['auto', 0]} ml={[0, 4]}>
              <button type="button" onClick={handleLogout}>
                Logout
              </button>
            </Box>
          )}
        </Box>

        {children && (
          <Box as={Media} lessThan="md">
            <Text fontWeight="normal">{children}</Text>
          </Box>
        )}
      </Box>
    </Text>
  );
};
MainMenu.propTypes = {
  // mobile footer
  children: PropTypes.any,
};

export default MainMenu;
