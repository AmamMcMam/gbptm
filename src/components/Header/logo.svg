<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 24.1.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 753.8 175.9" style="enable-background:new 0 0 753.8 175.9;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#ED3D63;}
	.st1{fill:#1B1464;}
</style>
<g>
	<g>
		<g>
			<path class="st0" d="M700.1,56c-18.3,0-33,14.9-33,33.2c0,31.7,33,73.1,33,73.1s33-41.4,33-73.1C733.1,70.9,718.3,56,700.1,56z
				 M700.1,105.4c-8.6,0-15.6-7-15.6-15.7s7-15.7,15.6-15.7c8.6,0,15.6,7,15.6,15.7C715.7,98.3,708.7,105.4,700.1,105.4z"/>
		</g>
		<g>
			<path class="st0" d="M715.7,30.4c0,8.7-7,15.7-15.6,15.7c-8.6,0-15.6-7-15.6-15.7c0-8.7,7-15.7,15.6-15.7
				C708.7,14.6,715.7,21.7,715.7,30.4z"/>
		</g>
	</g>
	<g>
		<path class="st1" d="M37.7,36.6h15.7v20.7c-2.5,0.8-4.9,1.4-7.2,1.8c-2.2,0.3-4.5,0.5-6.9,0.5c-6,0-10.5-1.8-13.7-5.3
			c-3.1-3.5-4.7-8.6-4.7-15.2c0-6.4,1.8-11.5,5.5-15.1c3.6-3.6,8.7-5.4,15.2-5.4c4.1,0,8,0.8,11.7,2.5l-2.8,6.8
			c-2.9-1.5-5.9-2.2-9-2.2c-3.6,0-6.5,1.2-8.7,3.7c-2.2,2.5-3.3,5.8-3.3,9.9c0,4.3,0.9,7.6,2.6,9.9c1.8,2.3,4.3,3.4,7.7,3.4
			c1.7,0,3.5-0.2,5.3-0.5v-8.3h-7.5V36.6z"/>
		<path class="st1" d="M71.1,43.7V59h-8.4V19.1h11.5c5.4,0,9.3,1,11.9,3s3.9,5,3.9,9c0,2.3-0.6,4.4-1.9,6.3
			c-1.3,1.8-3.1,3.3-5.4,4.3c5.9,9,9.8,14.8,11.6,17.4H85l-9.4-15.3H71.1z M71.1,36.8h2.7c2.6,0,4.6-0.4,5.9-1.3
			c1.3-0.9,1.9-2.3,1.9-4.2c0-1.9-0.6-3.2-1.9-4c-1.3-0.8-3.3-1.2-6-1.2h-2.5V36.8z"/>
		<path class="st1" d="M122,59H99.3V19.1H122V26h-14.4v8.8H121v6.9h-13.4V52H122V59z"/>
		<path class="st1" d="M154.4,59l-2.9-9.5h-14.4l-2.9,9.5h-9l13.9-40.1h10.2l14,40.1H154.4z M149.6,42.4c-2.6-8.6-4.1-13.5-4.5-14.6
			c-0.3-1.1-0.6-2-0.7-2.7c-0.6,2.3-2.3,8.1-5.1,17.3H149.6z"/>
		<path class="st1" d="M183.7,59h-8.4V26.1h-10.7v-7h29.8v7h-10.7V59z"/>
		<path class="st1" d="M214.9,19.1h12.3c5.6,0,9.7,0.8,12.2,2.4c2.5,1.6,3.8,4.2,3.8,7.7c0,2.4-0.6,4.3-1.7,5.9
			c-1.1,1.5-2.6,2.4-4.4,2.8v0.3c2.5,0.6,4.3,1.6,5.4,3.2c1.1,1.5,1.7,3.6,1.7,6.2c0,3.6-1.3,6.5-3.9,8.5c-2.6,2-6.1,3.1-10.6,3.1
			h-14.8V19.1z M223.2,34.9h4.9c2.3,0,3.9-0.4,4.9-1.1c1-0.7,1.5-1.9,1.5-3.5c0-1.5-0.6-2.6-1.7-3.3c-1.1-0.7-2.9-1-5.3-1h-4.4V34.9
			z M223.2,41.6V52h5.5c2.3,0,4-0.4,5.1-1.3c1.1-0.9,1.6-2.3,1.6-4.1c0-3.3-2.3-5-7-5H223.2z"/>
		<path class="st1" d="M260.4,43.7V59H252V19.1h11.5c5.4,0,9.3,1,11.9,3s3.9,5,3.9,9c0,2.3-0.6,4.4-1.9,6.3
			c-1.3,1.8-3.1,3.3-5.4,4.3c5.9,9,9.8,14.8,11.6,17.4h-9.3l-9.4-15.3H260.4z M260.4,36.8h2.7c2.6,0,4.6-0.4,5.9-1.3
			c1.3-0.9,1.9-2.3,1.9-4.2c0-1.9-0.6-3.2-1.9-4c-1.3-0.8-3.3-1.2-6-1.2h-2.5V36.8z"/>
		<path class="st1" d="M288.6,59V19.1h8.4V59H288.6z"/>
		<path class="st1" d="M322.1,59h-8.4V26.1H303v-7h29.8v7h-10.7V59z"/>
		<path class="st1" d="M338.9,59V19.1h8.4V59H338.9z"/>
		<path class="st1" d="M380.6,47.9c0,3.6-1.3,6.4-3.8,8.5c-2.6,2.1-6.1,3.1-10.7,3.1c-4.2,0-7.9-0.8-11.2-2.4v-7.9
			c2.7,1.2,4.9,2,6.8,2.5s3.5,0.7,5.1,0.7c1.8,0,3.2-0.4,4.2-1.1c1-0.7,1.5-1.8,1.5-3.2c0-0.8-0.2-1.5-0.7-2.1
			c-0.4-0.6-1.1-1.2-1.9-1.8c-0.8-0.6-2.5-1.5-5.1-2.7c-2.4-1.1-4.2-2.2-5.4-3.3c-1.2-1.1-2.2-2.3-2.9-3.7c-0.7-1.4-1.1-3-1.1-4.9
			c0-3.5,1.2-6.3,3.6-8.3c2.4-2,5.6-3,9.8-3c2.1,0,4,0.2,5.9,0.7c1.9,0.5,3.8,1.2,5.8,2.1l-2.7,6.6c-2.1-0.9-3.9-1.5-5.2-1.8
			c-1.4-0.3-2.7-0.5-4.1-0.5c-1.6,0-2.8,0.4-3.6,1.1c-0.8,0.7-1.3,1.7-1.3,2.9c0,0.7,0.2,1.4,0.5,2c0.3,0.6,0.9,1.1,1.6,1.6
			c0.7,0.5,2.5,1.5,5.3,2.8c3.7,1.8,6.2,3.6,7.6,5.4C379.9,43.1,380.6,45.3,380.6,47.9z"/>
		<path class="st1" d="M420.1,59h-8.4V41.7h-15.6V59h-8.4V19.1h8.4v15.6h15.6V19.1h8.4V59z"/>
		<path class="st1" d="M471.7,31.5c0,4.3-1.3,7.6-4,9.9c-2.7,2.3-6.4,3.4-11.3,3.4h-3.6V59h-8.4V19.1h12.6c4.8,0,8.4,1,10.9,3.1
			C470.5,24.3,471.7,27.4,471.7,31.5z M452.8,37.8h2.8c2.6,0,4.5-0.5,5.8-1.5c1.3-1,1.9-2.5,1.9-4.5c0-2-0.5-3.4-1.6-4.4
			c-1.1-0.9-2.8-1.4-5-1.4h-3.8V37.8z"/>
		<path class="st1" d="M511.3,19.1v25.8c0,2.9-0.7,5.5-2,7.8s-3.2,3.9-5.7,5.1c-2.5,1.2-5.4,1.8-8.8,1.8c-5.1,0-9-1.3-11.8-3.9
			c-2.8-2.6-4.2-6.2-4.2-10.8V19.1h8.4v24.4c0,3.1,0.6,5.3,1.8,6.8c1.2,1.4,3.3,2.2,6.1,2.2c2.7,0,4.7-0.7,6-2.2
			c1.2-1.4,1.8-3.7,1.8-6.8V19.1H511.3z"/>
		<path class="st1" d="M521,19.1h12.3c5.6,0,9.7,0.8,12.2,2.4c2.5,1.6,3.8,4.2,3.8,7.7c0,2.4-0.6,4.3-1.7,5.9
			c-1.1,1.5-2.6,2.4-4.4,2.8v0.3c2.5,0.6,4.3,1.6,5.4,3.2c1.1,1.5,1.7,3.6,1.7,6.2c0,3.6-1.3,6.5-3.9,8.5c-2.6,2-6.1,3.1-10.6,3.1
			H521V19.1z M529.4,34.9h4.9c2.3,0,3.9-0.4,4.9-1.1c1-0.7,1.5-1.9,1.5-3.5c0-1.5-0.6-2.6-1.7-3.3c-1.1-0.7-2.9-1-5.3-1h-4.4V34.9z
			 M529.4,41.6V52h5.5c2.3,0,4-0.4,5.1-1.3c1.1-0.9,1.6-2.3,1.6-4.1c0-3.3-2.3-5-7-5H529.4z"/>
		<path class="st1" d="M558.2,59V19.1h8.4V52h16v7H558.2z"/>
		<path class="st1" d="M589.5,59V19.1h8.4V59H589.5z"/>
		<path class="st1" d="M624.6,25.5c-3.2,0-5.6,1.2-7.3,3.6c-1.7,2.4-2.6,5.7-2.6,10c0,8.9,3.3,13.3,9.9,13.3c2.8,0,6.1-0.7,10.1-2.1
			v7.1c-3.2,1.4-6.9,2-10.9,2c-5.7,0-10.1-1.8-13.2-5.3c-3-3.5-4.6-8.6-4.6-15.2c0-4.2,0.7-7.8,2.2-10.9c1.5-3.1,3.6-5.5,6.4-7.2
			s6.1-2.5,9.8-2.5c3.8,0,7.7,0.9,11.6,2.8l-2.7,6.9c-1.5-0.7-3-1.3-4.5-1.9S626,25.5,624.6,25.5z"/>
	</g>
	<g>
		<path class="st1" d="M57.1,159.8H41.2V98.1H20.9V84.9h56.5v13.2H57.1V159.8z"/>
		<path class="st1" d="M156.9,122.2c0,12.4-3.1,21.9-9.2,28.6c-6.1,6.7-15,10-26.4,10c-11.5,0-20.3-3.3-26.4-10
			c-6.1-6.7-9.2-16.2-9.2-28.7c0-12.5,3.1-22,9.2-28.5c6.2-6.6,15-9.9,26.5-9.9c11.5,0,20.3,3.3,26.4,9.9
			C153.9,100.3,156.9,109.8,156.9,122.2z M102.3,122.2c0,8.4,1.6,14.7,4.8,18.9c3.2,4.2,7.9,6.3,14.2,6.3c12.7,0,19-8.4,19-25.2
			c0-16.9-6.3-25.3-18.9-25.3c-6.3,0-11.1,2.1-14.3,6.4S102.3,113.9,102.3,122.2z"/>
		<path class="st1" d="M172.4,159.8V84.9h15.9v74.9H172.4z"/>
		<path class="st1" d="M207.2,159.8V84.9H223v61.8h30.4v13.1H207.2z"/>
		<path class="st1" d="M309.5,159.8h-43.1V84.9h43.1v13h-27.2v16.4h25.4v13h-25.4v19.3h27.2V159.8z"/>
		<path class="st1" d="M354,159.8h-15.9V98.1h-20.3V84.9h56.5v13.2H354V159.8z"/>
		<path class="st1" d="M444.8,159.8l-18-58.7h-0.5c0.6,11.9,1,19.9,1,23.9v34.8h-14.2V84.9h21.6l17.7,57.3h0.3l18.8-57.3h21.6v74.9
			h-14.8v-35.4c0-1.7,0-3.6,0.1-5.8c0.1-2.2,0.3-8,0.7-17.4h-0.5l-19.3,58.6H444.8z"/>
		<path class="st1" d="M557.9,159.8l-5.4-17.8h-27.3l-5.4,17.8h-17.1L529,84.6h19.4l26.5,75.2H557.9z M548.6,128.6
			c-5-16.2-7.8-25.3-8.5-27.4c-0.6-2.1-1.1-3.8-1.4-5c-1.1,4.4-4.4,15.2-9.7,32.4H548.6z"/>
		<path class="st1" d="M636.1,108.3c0,8.1-2.5,14.2-7.6,18.5c-5,4.3-12.2,6.4-21.5,6.4h-6.8v26.6h-15.9V84.9h23.9
			c9.1,0,16,2,20.7,5.9C633.8,94.7,636.1,100.5,636.1,108.3z M600.3,120.1h5.2c4.9,0,8.5-1,11-2.9c2.4-1.9,3.6-4.7,3.6-8.4
			c0-3.7-1-6.5-3-8.2c-2-1.8-5.2-2.7-9.6-2.7h-7.2V120.1z"/>
	</g>
</g>
</svg>
