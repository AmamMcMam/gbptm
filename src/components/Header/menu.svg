<svg xmlns="http://www.w3.org/2000/svg" width="34" height="28" viewBox="0 0 34 28">
    <path fill="none" fill-rule="evenodd" stroke="#1B1464" stroke-width="2" d="M0 26.5h34M0 1.5h34m-34 12h34"/>
</svg>
