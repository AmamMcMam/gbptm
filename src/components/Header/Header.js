import React, { useState } from 'react';
import { Link } from 'react-router-dom';

import Box from '../Box';
import VisuallyHidden from '../VisuallyHidden';
import { Media } from '../Media';
import Drawer from '../Drawer';
import { faBars } from '@fortawesome/free-solid-svg-icons';

import Icon from '../Icon';

import Logo from './Logo';
import MainMenu from './MainMenu';

const Header = ({ children }) => {
  const [isMenuVisible, setIsMenuVisible] = useState(false);

  return (
    <Box
      as="header"
      display="flex"
      alignItems="center"
      px={[3, 4]}
      py={2}
      bg="white"
      minHeight={60}
    >
      <Box flexShrink={0}>
        <Link to="/">
          <Logo />
        </Link>
      </Box>

      <Box as="nav" width="100%" aria-labelledby="menu-main">
        <h2 id="menu-main">
          <VisuallyHidden>Main menu</VisuallyHidden>
        </h2>

        <Box as={Media} at="sm" display="flex" justifyContent="flex-end">
          <button
            type="button"
            aria-expanded={isMenuVisible}
            onClick={() => setIsMenuVisible(!isMenuVisible)}
          >
            <VisuallyHidden>Toggle main menu</VisuallyHidden>
            <Icon icon={faBars} size="2x" />
          </button>

          <Drawer visible={isMenuVisible} zIndex={200}>
            <MainMenu children={children} />
          </Drawer>
        </Box>

        <Media greaterThan="sm">
          <MainMenu children={children} />
        </Media>
      </Box>
    </Box>
  );
};

export default Header;
