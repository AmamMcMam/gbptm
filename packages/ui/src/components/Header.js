import React, { Component } from 'react';
import { Link } from 'react-router-dom';

import styles from './css/header.module.css';
import logo from '../images/logo.svg';

class Header extends Component {
  render() {
    return (
      <header className={styles.header}>
        <Link to="/" title="Go to home page" className={styles.logoWrapper}>
          <img
            src={logo}
            alt="'The Great British Toilet Map' logo"
            className={styles.logo}
          />
          <span className={styles.title}>
            The Great British <span>Public Toilet Map</span>
          </span>
        </Link>
        <a
          href="http://web.rca.ac.uk/greatbritishpublictoiletmap"
          className={styles.donate}
          target="_blank"
          rel="noopener noreferrer"
        >
          Give a pound to spend a penny!
        </a>
      </header>
    );
  }
}

export default Header;
