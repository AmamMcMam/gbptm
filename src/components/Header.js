import React from 'react';
import styled from '@emotion/styled';
import { Link, withRouter } from 'react-router-dom';

import logo from '../images/logo.svg';

import Box from './Box';
import Text from './Text';

const Logo = styled((props) => (
  <img {...props} src={logo} alt="The Great British Toilet Map" />
))`
  height: 2rem;
`;

// Todo: Contact link
// Todo: Update logo
const Header = () => {
  return (
    <Box
      as="header"
      display="flex"
      alignItems="center"
      px={4}
      py={2}
      bg="white"
      color="primary"
      minHeight="60px"
    >
      <Link to="/">
        <Logo />
      </Link>

      <Text fontWeight="bold" width="100%">
        <nav>
          <Box as="ul" display="flex">
            <Box as="li" ml={4}>
              <Link to="/">Find Loo</Link>
            </Box>
            <Box as="li" ml={4}>
              <Link to="/report">Add Loo</Link>
            </Box>

            <Box as="li" ml="auto">
              <Link to="/About">About</Link>
            </Box>
            <Box as="li" ml={4}>
              <Link to="/use-our-loos">Our Sponsor</Link>
            </Box>
            <Box as="li" ml={4}>
              <Link to="">Contact</Link>
            </Box>
          </Box>
        </nav>
      </Text>
    </Box>
  );
};

export default withRouter(Header);
