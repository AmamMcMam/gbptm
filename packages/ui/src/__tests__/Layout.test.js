import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../components/Header';
import Footer from '../components/Footer';

describe('Layout', () => {
  // Jest snapshot
  it('renders Header', () => {
    var tree = renderer.create(<Header />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  // Jest snapshot
  it('renders Footer', () => {
    var tree = renderer.create(<Footer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
