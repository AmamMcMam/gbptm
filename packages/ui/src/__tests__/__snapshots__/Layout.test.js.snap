// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders Footer 1`] = `
<footer
  className="footer"
>
  <ul
    className="footerList"
  >
    <li
      className="footerListItem"
    >
      <a
        className="footerItem"
        onClick={[Function]}
        style={Object {}}
      >
        <span
          className="footerItemText"
        >
          My toilet preferences
        </span>
      </a>
    </li>
    <li
      className="footerListItem"
    >
      <a
        className="footerItem"
        onClick={[Function]}
        style={Object {}}
      >
        <span
          className="footerItemText"
        >
          About this project
        </span>
      </a>
    </li>
  </ul>
</footer>
`;

exports[`Layout renders Header 1`] = `
<header
  className="header"
>
  <a
    className="logoWrapper"
    onClick={[Function]}
    style={Object {}}
    title="Go to home page"
  >
    <img
      alt="'The Great British Toilet Map' logo"
      className="logo"
      src="logo.svg"
    />
    <span
      className="title"
    >
      The Great British 
      <span>
        Public Toilet Map
      </span>
    </span>
  </a>
  <a
    className="donate"
    href="http://web.rca.ac.uk/greatbritishpublictoiletmap"
    rel="noopener noreferrer"
    target="_blank"
  >
    Give a pound to spend a penny!
  </a>
</header>
`;
